tests/pcap/runpcaptests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+./torq.sh start discovery1
+./torq.sh start tickerplant1 
+./torq.sh start rdb1 -extras -.rdb.replaylog 0
+/usr/bin/rlwrap q torq.q -load ${KDBCODE}/processes/filealerter.q -proctype filealerter -procname filealerter1 -test ${KDBTESTS}/pcap -debug -.fa.decodepcaps 1 -.fa.polltime 00:10 -.servers.CONNECTIONS rdb -.fa.alreadyprocessed ${KDBTESTS}/pcap/filealerterprocessed -.fa.inputcsv ${KDBTESTS}/pcap/filealerter.csv
```

Comment:
this hardcoded path for `rlwrap` might break depending on specific set up of linux box where script is run

Reply:
should it be replaced with `${RLWRAP}?`

Reply:
I think that should work assuming `setenv.sh` has been sourced; might also be OK to just do `rlwrap` & assume it'll be on the path (this is what you're doing with `q` anwyay by the looks of it)

Reply:
ah okay thanks, changed it to `rlwrap`